### PR TITLE
Feature/example/redux-saga-typescript-graphql-fluentui

### PR DESCRIPTION
**Pull request by bowen31337**
this is another nexjs example which is placed in the examples folder.

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 12s | 12.3s | ⚠️ +256ms |
| nodeModulesSize | 66.1 MB | 66.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.966 | 2.005 | ⚠️ +0.04 |
| / avg req/sec | 1271.61 | 1247.01 | ⚠️ -24.6 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.225 | 1.18 | -0.05 |
| /error-in-render avg req/sec | 2040.72 | 2119.17 | +78.45 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..18e4.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.71 kB | 5.71 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.09 kB | 7.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 958 B | 958 B | ✓ |
| withRouter.html gzip | 945 B | 945 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 13.1s | 13.2s | ⚠️ +41ms |
| nodeModulesSize | 66.1 MB | 66.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..18e4.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.71 kB | 5.71 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.09 kB | 7.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _error.js | 876 kB | 876 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 915 kB | 915 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: daca4d0df6c75056f4c8e79cc9ad96f637c185fd

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 11.3s | 10.7s | -598ms |
| nodeModulesSize | 66.1 MB | 66.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.974 | 1.961 | -0.01 |
| / avg req/sec | 1266.31 | 1274.58 | +8.27 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.377 | 1.384 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 1815.48 | 1806.52 | ⚠️ -8.96 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..18e4.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.71 kB | 5.71 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.09 kB | 7.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 958 B | 958 B | ✓ |
| withRouter.html gzip | 945 B | 945 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 12.2s | 13.4s | ⚠️ +1.2s |
| nodeModulesSize | 66.1 MB | 66.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..18e4.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.71 kB | 5.71 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.09 kB | 7.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _error.js | 876 kB | 876 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 915 kB | 915 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: eff6423d978872fa29d0cc6febb0ca04577b5204

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 12.4s | 14.1s | ⚠️ +1.7s |
| nodeModulesSize | 66.1 MB | 66.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.016 | 1.93 | -0.09 |
| / avg req/sec | 1240.33 | 1295.02 | +54.69 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.279 | 1.199 | -0.08 |
| /error-in-render avg req/sec | 1954.84 | 2084.7 | +129.86 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..c646.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.71 kB | 5.71 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 960 B | 960 B | ✓ |
| withRouter.html gzip | 946 B | 946 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 13.4s | 13.7s | ⚠️ +247ms |
| nodeModulesSize | 66.1 MB | 66.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..c646.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.71 kB | 5.71 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _error.js | 876 kB | 876 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 877 kB | 877 kB | ✓ |
| link.js | 915 kB | 915 kB | ✓ |
| routerDirect.js | 909 kB | 909 kB | ✓ |
| withRouter.js | 909 kB | 909 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: eb56c60871d997763a9c921ab465b51f79d2dc39

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 11.8s | 11.8s | ⚠️ +66ms |
| nodeModulesSize | 66.1 MB | 66.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.954 | 1.976 | ⚠️ +0.02 |
| / avg req/sec | 1279.62 | 1264.95 | ⚠️ -14.67 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.195 | 1.218 | ⚠️ +0.02 |
| /error-in-render avg req/sec | 2091.94 | 2052.2 | ⚠️ -39.74 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..c646.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.71 kB | 5.71 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 960 B | 960 B | ✓ |
| withRouter.html gzip | 946 B | 946 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| buildDuration | 12.6s | 13.5s | ⚠️ +836ms |
| nodeModulesSize | 66.1 MB | 66.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..c646.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.71 kB | 5.71 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bowen31337/next.js feature/example/redux-saga-typescript-graphql-fluentui | Change |
| - | - | - | - |
| _error.js | 876 kB | 876 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 877 kB | 877 kB | ✓ |
| link.js | 915 kB | 915 kB | ✓ |
| routerDirect.js | 909 kB | 909 kB | ✓ |
| withRouter.js | 909 kB | 909 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: 6de6d46f39ff061eff3d6705f20e0d5461afa345

**Comment by bowen31337** [comment]
@Timer @lfades @chibicode @timneutkens please let me know if there are any concerns regarding this PR.

File: examples/redux-saga-typescript-graphql-fluentui/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# redux-saga-typescript-graphQL-fluentUI example
+
```
Comment (lfades) [inline examples/redux-saga-typescript-graphql-fluentui/README.md:2]: Can you please add more details about the example here under the title 🙏 , what it does, what libraries is it using, why is it using them and anything else that may be important to know about the example.

File: examples/redux-saga-typescript-graphql-fluentui/pages/_app.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    if (ctx.req) {
+      ctx.store.dispatch(END)
+      await (ctx.store as SagaStore).sagaTask?.toPromise()
+    }
+
+    // 3. Return props
+    return {
+      pageProps,
+    }
+  }
```
Comment (lfades) [inline examples/redux-saga-typescript-graphql-fluentui/pages/_app.tsx:27]: We are currently not taking examples using `getInitialProps` for simple stores, take a look at our other Redux examples to see how it works without it, e.g: https://github.com/vercel/next.js/tree/canary/examples/with-redux-saga

File: examples/redux-saga-typescript-graphql-fluentui/constants/env.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/**
+ * Environment variables
+ */
+export const Env = {
+  NODE_ENV: process.env.NODE_ENV,
+  API_GATEWAY_ENTRYPOINT:
+    process.env.API_GATEWAY_ENTRYPOINT || 'http://localhost:9000',
+}
```
Comment (lfades) [inline examples/redux-saga-typescript-graphql-fluentui/constants/env.ts:8]: Let's try to remove this file, it's better to use environment variables directly

File: examples/redux-saga-typescript-graphql-fluentui/store/sagas.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+//         landSize,
+//         floorSize,
+//         address {
+//           streetType,
+//           formattedAddress,
+//           postCode,
+//           suburb
+//         }
+//     }
+// }`
```
Comment (lfades) [inline examples/redux-saga-typescript-graphql-fluentui/store/sagas.ts:19]: What's this for?

**Review by lfades — CHANGES_REQUESTED** [review]
Hi there @bowen31337 Thank you for the PR! I did a review and left some comments.

File: examples/redux-saga-typescript-graphql-fluentui/README.md (hunk 1 above)
Comment (bowen31337) [reply examples/redux-saga-typescript-graphql-fluentui/README.md:2]: yes, i will add more details

**Pull request closed without merging**